playground/src/main/res/layout/bottom_sheet_output.xml

```diff
@@ -55,21 +51,79 @@
                 android:layout_height="wrap_content"
                 android:orientation="vertical">
 
+                <LinearLayout
+                    android:id="@+id/errorLayout"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_margin="16dp"
+                    android:background="@drawable/error_round_background"
+                    android:orientation="vertical">
+
+                    <ImageView
+                        android:id="@+id/errorImage"
+                        android:layout_width="match_parent"
+                        android:layout_height="80dp"
+                        android:layout_marginTop="8dp"
+                        android:src="@drawable/ic_importerror"></ImageView>
+
+                    <TextView
+                        android:id="@+id/errorTextExample"
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        style="@style/ConsoleOutput"
+                        android:textColor="@color/error_text"
+                        android:layout_marginTop="16dp"
+                        android:text="Couldn't find an object with name 'cube' in the 'math' library to import."></TextView>
+
+                </LinearLayout>
+
+                <TextView
+                    android:id="@+id/errorTextTip"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    style="@style/ConsoleOutput"
+                    android:background="@drawable/error_tip_round_backgrond"
+                    android:layout_marginLeft="16dp"
+                    android:layout_marginRight="16dp"
+                    android:padding="16dp"
```

Comment:
Please make sure to use spacing units from dimens file where ever you are using them for paddings and margins. Sizes can either be hardcoded or they can be added as separate tokens in dimens file.

Reply:
Done